python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 1.4.1
+
+### Feature:
+- Implement `check_table` Operator to validate data quality at table level [#1239](https://github.com/astronomer/astro-sdk/pull/1239)
```

Comment:
Should we also add a PR for the documentation here for users?

Reply:
We have added that I docs section